gem/client/css/modules/_m-comments.scss

```diff
@@ -187,6 +187,10 @@ a.report {
   text-align: right;
   color: #A3A3A3;
 }
+.comment__reply-meta-replies-list-disabled {
+  margin: 0;
+  color: #A3A3A3;
+}
```

Comment:
hhmm scope of work design still have that, which makes sense to me because it is still a lin functionality with 0 reply comments to go to.